src/ngx_http_lua_output.c

```diff
@@ -802,4 +802,100 @@ ngx_http_lua_flush_cleanup(void *data)
     ctx->flushing_coros--;
 }
 
+#ifndef NGX_LUA_NO_FFI_API
+int
+ngx_http_lua_ffi_write(ngx_http_request_t *r, const char* str, size_t offset, size_t len)
```

Comment:
Style: `const char* str` should be `const char *str`. Be careful. Always be consistent with the rest of the code base.